Dockerfile

```diff
@@ -1,13 +1,11 @@
-FROM node:22-slim
-
-RUN npm install -g pnpm
+FROM oven/bun:1.3.5-alpine
 
 WORKDIR /app
 
-COPY package.json pnpm-lock.yaml ./
-RUN pnpm install --frozen-lockfile
+COPY package.json bun.lock ./
+RUN bun install --frozen-lockfile
 
 COPY . .
```

Comment:
I believe it would be much better to fix this statement instead of adding a .dockerignore file.